guides/source/configuring.md

```diff
@@ -140,6 +140,8 @@ defaults to `:debug` for all environments. The available log levels are: `:debug
 
 * `secrets.secret_key_base` is used for specifying a key which allows sessions for the application to be verified against a known secure key to prevent tampering. Applications get `secrets.secret_key_base` initialized to a random key present in `config/secrets.yml`.
 
+* `config.secrets_cipher` allows you to change the cipher used to protect the `config/secrets.yml` file. This defaults to `"aes-128-gcm"`. Versions of OpenSSL earlier than 1.0.01, unfortunately, don't support GCM mode, so degrading this to `"aes-128-cbc"` may be necessary. If you find yourself in this situation, strongly consider updating OpenSSL to a modern version if at all possible rather than weakening this setting.
```

Comment:
Speaking as an infosec person, I don't think that's a reasonable position to take.

API design matters. It doesn't matter how smart and capable your users are if your APIs guide users to make the wrong choice. Naming a file `secrets.yml` and putting things that look like secrets into them is sending people the wrong message, and users — smart, capable users — *will* misunderstand its purpose and make mistakes. In this case, security-impacting mistakes.

On the flip side, what's the actual cost of changing the design of this feature to guide people away from making mistakes? My guess is: not much.

You're basically arguing in favor of this:
![ityr0voz1qrpqdhkxraul0oh4juqljmnzq-kfjmwr8o](https://cloud.githubusercontent.com/assets/4338/23481070/ffc62ffe-fe7e-11e6-988f-e4aefa0e04c4.jpg)